src/Http/Http.Extensions/test/Microsoft.AspNetCore.Http.Extensions.Tests.csproj

```diff
@@ -16,6 +16,7 @@
     <Reference Include="Microsoft.AspNetCore.Http" />
     <Reference Include="Microsoft.AspNetCore.Http.Results" />
     <Reference Include="Microsoft.AspNetCore.Http.Extensions" />
+    <Reference Include="Microsoft.AspNetCore.Mvc.Core" />
```

Comment:
???

Reply:
We need the ```[FromQuery]``` attributes. The test cases basically copy the dependencies of the unit tests project that they are defined within.

Reply:
We'll sometimes copy a test helper class between projects, or move it to `src/Shared`, but I don't think it's worth blocking the PR on that. Maybe as a follow-up.